.changeset/kind-foxes-begin.md

```diff
@@ -11,3 +11,4 @@ Refactored `VideoPlayer` component to make it more modular and customisable.
   - Where you previously passed `showTitle={false}`, you should now pass `visuallyHiddenTitle={true}`.
 - The `branding` prop has been renamed to `showBranding`.
 - Individual video controls can be optionally hidden by setting any of the `showPlayPauseButton`, `showSeekControl`, `showCCButton`, `showMuteButton`, `showVolumeControl`, and `showFullScreenButton` props to `false`.
+- A custom play icon can be provided using the `playIcon` prop.
```

Comment:
@rezrah Am I alright updating the existing changeset like this?

If PB gets published before this is merged I'd be happy to add a new changeset

Reply:
Yep, this is fine.. 👍